StrangeCalculatorTask/src/main/java/Singleton.java

```diff
@@ -0,0 +1,38 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Singleton {
```

Comment:
Singleton is a name of the pattern, which you are using in this class. Please, rename this class corresponding to functionality, for example: 'Console' or 'Terminal'